### PR TITLE
Incrementer smart contract.

### PR DESCRIPTION
**Pull request by nicolad**
Example is based on this: https://github.com/paritytech/ink-examples/tree/main/incrementer

***
here is a short demo

https://user-images.githubusercontent.com/1477453/228445311-9d40d9ed-5e88-4346-b621-870bb40c311e.mp4

### CODE REVIEW

File: packages/frontend/src/components/incrementer/Increment.tsx [hunk 1]
```diff
@@ -7,109 +18,94 @@ import {
...
-      toast.success(`Successfully updated greeting`)
+      const incBy = form.values.incBy
+      await contractTx(api, activeAccount.address, contract, 'inc', {}, [incBy])
+      toast.success(`Successfully incremented`)
       form.reset()
-    } catch (e) {
-      console.error(e)
-      toast.error('Error while updating greeting. Try again.')
+    } catch (e: any) {
+      console.log(e)
```
Comment (nicolad) [inline packages/frontend/src/components/incrementer/Increment.tsx:65]: @wottpal any idea how to parse this error?

image

Comment (nicolad) [reply packages/frontend/src/components/incrementer/Increment.tsx:65]: once I will get response will followup with another MR

Comment (wottpal) [reply packages/frontend/src/components/incrementer/Increment.tsx:65]: sorry, having a lot of on-site events happening this week. will follow-up end of week or beginning of next one :) thanks for your efforts already!

Comment (nicolad) [reply packages/frontend/src/components/incrementer/Increment.tsx:65]: thx for the update

**Pull request merged**